projects/igniteui-angular/schematics/utils/dependency-handler.ts

```diff
@@ -4,7 +4,34 @@ import { getWorkspace } from '@schematics/angular/utility/config';
 import { Options } from '../interfaces/options';
 import { WorkspaceProject, ProjectType, WorkspaceSchema } from '@schematics/angular/utility/workspace-models';
 
+
+export enum PackageTarget {
+    DEV = 'devDependencies',
+    REGULAR = 'dependencies',
+    CUSTOM = 'custom'
+}
+interface PackageEntry {
+    name: string;
+    target: PackageTarget;
+}
+
+
 const schematicsPackage = '@igniteui/angular-schematics';
+/**
+ * Dependencies are explicitly defined here, so we avoid adding
+ * unnecessary packages to the consuming project's deps (ie the ones needed
+ * for the watermark)
+ */
+export const explicitDependencies: PackageEntry[] = [
+        { name: 'hammerjs', target: PackageTarget.REGULAR },
+        { name: 'jszip', target: PackageTarget.REGULAR },
+        { name: 'resize-observer-polyfill', target: PackageTarget.REGULAR },
+        { name: 'minireset.css', target: PackageTarget.CUSTOM },
+        { name: 'web-animations-js', target: PackageTarget.CUSTOM },
```

Comment:
right now as peer dependency `web-animations-js` is actually `REGULAR`, no? I see it's logic is separate, but still in case  it's made generic in the future